site/src/component/QuarterTooltip/CourseQuarterIndicator.tsx

```diff
@@ -123,7 +123,7 @@ const CourseQuarterIndicator: FC<CourseQuarterIndicatorProps> = (props) => {
 
               {termsInOrder.includes(`${prevYear} Spring`) && (
                 <span>
-                  <span className={emojiSize}>🌸</span>
+                  <span className={emojiSize}>🌱</span>
```

Comment:
I don't think changing this emoji "to get rid of the pink color" is the right move here. If it's just for that, we should instead just replace the course offerings tooltip chart with the table as seen in recent offerings (since it's something we discussed anyways).

I suggest reverting this emoji change for the PR, since it may introduce a change that existing users would have to get used to.